docs: Add first partial documentation site with docz (#58)
Bojagi/pablo: src/Input/Input.mdx

---
name: Input
route: /input
menu: Components
---

import { Props, Playground } from 'docz'
import { Box } from '../Box/Box.tsx'
import { Input } from './Input.tsx'
import { IconButton } from '../IconButton/index.ts'
import { Eye, EyeOff, Search } from 'react-feather';

# Input

Form element that allows users to type in single line text. For multi-line text you can use the `TextArea` component.

## Examples

### Simple Input

> The simples form of an `Input` is just the field itself. You can just pass a `useState` setter function to update state

Reviewer: ```suggestion
The simplest form of an `Input` is just the field itself. You can pass a `useState` setter function to update state
```

as the first argument of `onChange` is the value itself. 
The second argument is the full `onChange` event of the native `input` element.

<Playground>
  {() => {
    const [value, setValue] = React.useState('');
    return (
      <Input value={value} onChange={setValue} />
    );
  }}
</Playground>

### Input with label

> A simple input field doesn't really make sense as the context is missing. You can pass the `label` prop with the

Reviewer: I would remove the quite negative opinion here and stay more  neutral (doesn't really make sense)

```suggestion
Often a simple input field lacks context. You can pass the `label` prop with the
```
